chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -51,19 +51,21 @@
   subscribes :run, resources(:cookbook_file=> disable_rp_filter_file), :delayed
 end
 
-
 case neutron[:neutron][:networking_plugin]
```

Comment:
How about turning this into an ```if neutron[:neutron][:networking_plugin] == "ml2"```? Currently ```ml2``` is really the only ```when``` branch here as e.g. ```vmware``` doesn't have any l2 agent.